<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitConverterDialog</class>
 <widget class="QDialog" name="UnitConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单位转换器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="unitTypesLabel">
     <property name="text">
      <string>单位类型</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>值:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string>结果:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="copyPushButton">
     <property name="text">
      <string>复制结果</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="convertToLabel">
     <property name="text">
      <string>转换为:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="resultLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="unitTypesComboBox"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="convertToComboBox"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="UtQtUnitValueEdit" name="valueLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtUnitValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>unitTypesComboBox</tabstop>
  <tabstop>convertToComboBox</tabstop>
  <tabstop>valueLineEdit</tabstop>
  <tabstop>resultLineEdit</tabstop>
  <tabstop>copyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
